File: executions/regex.go
package main
import (
    "fmt"
    "regexp"
)

func main() {
    text := "any text here 1 2 3 4 5"
    
    regexLetters, _ := regexp.Compile("[a-f]")
    
    fmt.Println(regexLetters.FindAllString(text, 10)) //Return [a e e e]
    fmt.Println(regexLetters.ReplaceAllString(text, "*")) //Return "*ny t*xt h*r* 1 2 3 4 5
    fmt.Println(regexLetters.Split(text, -1)) // Return [ ny t xt h r  1 2 3 4 5] 
}

File: executions/regex.py
import re

text = 'any text here 1 2 3 4 5'
regexLetters = "[a-f]"

print(re.findall(regexLetters, text)) #Return ['a', 'e', 'e', 'e']

valueRegEx = (re.search(regexLetters, text))
print(valueRegEx.start()) # Return 0
print(re.sub(regexLetters, '*', text)) # return *ny t*xt h*r* 1 2 3 4 5
print(re.split(regexLetters, text)) # return ['', 'ny t', 'xt h', 'r', ' 1 2 3 4 5']